superset/assets/src/visualizations/leaflet_map.css

```diff
@@ -27,3 +27,29 @@
     width:auto !important;
     background:white !important
 }
+
+
+
+#heat-map-color-legend{
+  margin-top: 30px;
+}
+
+.legend-div{
+  width: 20px;
+  height:10px;
+  margin-left: 2px;
+}
+
+.horizontal{
+  display: flex;
+}
+.horizontal .legend-div{
+  margin-left: 0px;
+  margin-top: 4px;
+}
+
+.percentage-value{
+  width: 20px;
+  text-align: center;
+  margin-left: 2px;
+}
```

Comment:
style name should have prefix  to distinct from default styles names

Reply:
done